src/main/resources/db/migration/V0.5.26__remove_trait_abbreviations.sql

```diff
@@ -0,0 +1,18 @@
+/*
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ ALTER TABLE trait DROP COLUMN abbreviations ;
```

Comment:
I just merged BI-1201 with a migration labeled V0.5.26 (V0.5.26__add-program-key.sql), so will likely have to increment this number. I do also have a V0.5.27__add_germplasm_template_system_mapping.sql in my PR for BI-1176, so I could adjust that to 28 or this could be 28 depending on merge order?

Reply:
Merged BI-1176, so this would need to be 28